src/Components/Comments.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from "react";
+import {
+  getCommentsByReviewId,
+  getUsers,
+  avatarUrl,
+  commentAuthor,
+} from "../utils";
+
+export const Comments = ({ review_id }) => {
+  const [comments, setComments] = useState([]);
+  const [users, setUsers] = useState([]);
+  const [isCommentsLoading, setIsCommentsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsCommentsLoading(true);
+    getCommentsByReviewId(review_id).then((data) => {
+      setIsCommentsLoading(false);
+      const commentsData = data.comments;
+      setComments(commentsData);
+    });
+
+    getUsers().then((data) => {
+      const usersData = data.users;
+      setUsers(usersData);
+    });
+  }, [review_id]);
+
+  return review_id ? (
```

Comment:
You shouldn't need to check if there is a review id. Only if there is a review id will the router render this component.